src/framework/dropdown/_dropdown.scss

```diff
@@ -17,31 +17,31 @@
  */
 .dropdownLabel {
   @include baseDropdownLabel;
-  text-overflow: ellipsis; /* 1 */
-  overflow: hidden; /* 1 */
+  background-color: #f6fafc;
```

Comment:
I wanted to extract all colors anyway in a separate PR. As I reused one specific color I extracted just that one.